backend/audit/signals.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from .models import SingleAuditChecklist
+from support import cog_over
+
+
+@receiver(post_save, sender=SingleAuditChecklist)
+def fac_post_process(sender, instance, created, **kwargs):
```

Comment:
Wondering about whether this signal can/should live in the `support` app, so that all of the logic associated with `cog` assignment lives over there and nothing in `audit` needs to be responsible for triggering `cog` processing

Reply:
Suggestion accepted and implemented